great_expectations/expectations/expectation.py

```diff
@@ -46,28 +40,8 @@
     ExpectationConfiguration,
     parse_result_format,
 )
-from great_expectations.core.expectation_diagnostics.expectation_diagnostics import (
-    ExpectationDiagnostics,
-)
-from great_expectations.core.expectation_diagnostics.expectation_test_data_cases import (
-    ExpectationLegacyTestCaseAdapter,
-    ExpectationTestCase,
-    ExpectationTestDataCases,
-    TestBackend,
-    TestData,
-)
-from great_expectations.core.expectation_diagnostics.supporting_types import (
-    AugmentedLibraryMetadata,
-    ExpectationBackendTestResultCounts,
-    ExpectationDescriptionDiagnostics,
-    ExpectationDiagnosticMaturityMessages,
-    ExpectationErrorDiagnostics,
-    ExpectationExecutionEngineDiagnostics,
-    ExpectationMetricDiagnostics,
-    ExpectationRendererDiagnostics,
-    ExpectationTestDiagnostics,
-    Maturity,
-    RendererTestDiagnostics,
```

Comment:
Trying to slowly remove dependencies on `core` - this bundles everything together in one place for now